packages/key4hep-stack/package.py

```diff
@@ -244,11 +244,11 @@ class Key4hepStack(BundlePackage, Key4hepPackage):
 
     ############################## cepcsw #################
     #######################################################
-    #depends_on("cepcsw")
-    #k4_add_latest_commit_as_dependency("cepcsw", "cepc/cepcsw", when="@master")
+    depends_on("cepcsw")
```

Comment:
@mirguest I think the error is because here you either need to depend on `cepcsw@master` or release a version 0.2.x in order for the right k4fwcore to be picked up

Reply:
@vvolkl Thanks Valentin, I create a new release v0.2.0 for CEPCSW